src/controllers/trainee/routes.ts

```diff
@@ -1,10 +1,11 @@
 import * as express from "express";
 import ControllerTrainee from "./Controller";
 import { validateHandler } from "../../libs/routes";
+import {authMiddleware} from "../../libs/routes";
 import validConfigData from "./validate";
 const traineeRouter = express.Router();
 traineeRouter
-    .get("/", validateHandler(validConfigData.get), ControllerTrainee.get)
+    .get("/", validateHandler(validConfigData.create), authMiddleware('user1', 'read'), ControllerTrainee.get)
```

Comment:
I believe get method doesn't have a body and you are doing all of your validations to the body.
please rearrange your middlewares so that request goes through authMiddleware first.